frontend/src/views/DrivesView.vue

```diff
@@ -114,13 +126,17 @@ export default {
   methods: {
     ...mapActions(namespaces.drives, [apiActions.fetchDrives]),
     showDrive(id) {
-      this.driveVisible = id;
+      this.driveVisible = this.driveVisible === id ? null : id;
```

Comment:
I think there was a separate bug for that 👍

Reply:
I'm not sure whether there is an issue related to that. I just wanted to make it more toggle-like than only openable.